script.js

```diff
@@ -1,28 +1,31 @@
-const msgWaiting = document.querySelector(".message-waiting");
-const searchForm = document.getElementById("search-form");
-const booksDisplay = document.getElementById("books-display");
+const $msgWaiting = $(".message-waiting");
+const $searchForm = $("#search-form");
+const $booksDisplay = $("#books-display");
 
 const googleBooksURL = "https://www.googleapis.com/books/v1/volumes?q=";
 const apiKey = "&key=AIzaSyCw6aeDwd2srAlaiVBFSSBvyl2O7atCTaA";
 
-searchForm.addEventListener("submit", () => {
-  msgWaiting.style.display = "none";
-  const searchInput = document.getElementById("search-input").value;
+$searchForm.on("submit", () => {
+  $msgWaiting.hide();
```

Comment:
jQuery's `hide` function more-or-less will do the same thing as setting the display property to `none`, only it's a bit more clever based on the initial display property of the element(s) being hidden.